btax/param_defaults/btax_defaults.json

```diff
@@ -0,0 +1,559 @@
+[
+  [
+    "btax_betr_corp", 
+    {
+      "long_name": "", 
+      "notes": "notes", 
+      "description": "description", 
+      "value": [
+        [
+          0.986
+        ]
+      ]
+    }
+  ], 
+  [
+    "btax_betr_entity_Switch", 
+    {
+      "long_name": "", 
+      "notes": "notes", 
+      "description": "description", 
+      "value": [
+        [
+          0.986
```

Comment:
Each "value" field should be a list that is one element long where the one element contains the default.